packages/web/src/components/spa/Footer/Footer.jsx

```diff
@@ -12,30 +12,37 @@ export const Footer = ({ colour, iconcolor }) => {
             </div>
             <div className="social-media">
                 <div className="links">
-                    <a
-                        href="https://www.facebook.com/TheHubAUBG"
-                        target="_blank"
-                    >
-                        <FaFacebookSquare
+                    <span className="icon-footer">
```

Comment:
Instead of an anchor element,bind the function for opening links from Global.js to an onClick on the span component.